nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -76,6 +76,12 @@ def wrap_operator(operator, operator_info: PatchedOperatorInfo):
 
     @functools.wraps(operator)
     def wrapped(*args, **kwargs):
+        from nncf.torch.dynamic_graph.patch_pytorch import operators_are_wrapped
```

Comment:
Please think about how to implement it without conditional import.

Reply:
It will be possible if we create a separate file `patch_pytorch_utils.py` and put there some constants from `patch_pytorch.py` such as `_OPERATORS_ALREADY_WRAPPED`. Then both `patch_pytorch.py` and `wrappers.py` will import from `patch_pytorch_utils.py` as usual. Is that better?